psydac/feec/multipatch/tests/tests_feec_system_multipatch_2d.py

```diff
@@ -0,0 +1,174 @@
+import pytest
+import numpy as np
+from mpi4py import MPI
+
+from sympy import Matrix
+
+from sympde.calculus      import grad, dot, curl
+from sympde.calculus      import minus, plus, cross
+from sympde.topology      import Derham
+from sympde.topology      import elements_of
+from sympde.topology      import NormalVector
+from sympde.topology      import Square
+from sympde.topology      import IdentityMapping, PolarMapping
+from sympde.expr.expr     import LinearForm, BilinearForm
+from sympde.expr.expr     import integral
+from sympde.expr.expr     import Norm
+from sympde.expr.equation import find, EssentialBC
+
+from psydac.feec.multipatch.api import discretize  # TODO: when possible, use line above
+
+from psydac.linalg.iterative_solvers import cg, pcg
+from psydac.linalg.utilities         import array_to_stencil
+
+from psydac.fem.basic          import FemField
+from psydac.feec.pull_push     import push_2d_hcurl, pull_2d_hcurl
+
+from psydac.feec.multipatch.fem_linear_operators import FemLinearOperator, IdLinearOperator
+from psydac.feec.multipatch.fem_linear_operators import SumLinearOperator, MultLinearOperator, ComposedLinearOperator
+from psydac.feec.multipatch.operators import BrokenMass
+from psydac.feec.multipatch.operators import ConformingProjection_V1
+
+from scipy.sparse.linalg import cg
+comm = MPI.COMM_WORLD
+#==============================================================================
+def run_conga_maxwell_2d(uex, f, alpha, domain, ncells, degree, tol, use_scipy=True):
```

Comment:
again, these tests are not using the new feec-multipatch interface, so I am not really convinced that they are relevant at this point

Reply:
yes, they are no longer relevant